src/__tests__/Home.test.js

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+import { BrowserRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Home from '../components/Home';
+import store from '../redux/store';
+
+describe('Home', () => {
+  it('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <BrowserRouter>
+          <Provider store={store}>
+            <Home />
+          </Provider>
+        </BrowserRouter>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
[OPTIONAL] The tests for some of the components are failing as shown below :point_down: . Kindly check and ensure that they are all working and passing to ensure the best and proffessional website. :+1: 

![Screenshot from 2023-02-08 00-15-02](https://user-images.githubusercontent.com/65233860/217397433-7d3f51f7-0d4d-4baf-b3d3-7c3aa4836318.png)